packages/web-components/src/components/ic-pagination-item/ic-pagination-item.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Component,
+  Prop,
+  Element,
+  h,
+  Event,
+  EventEmitter,
+} from "@stencil/core";
+import { IcPaginationItemType } from "./ic-pagination-item.types";
+import { IcThemeForeground } from "../../interface";
+
+@Component({
+  tag: "ic-pagination-item",
+  styleUrl: "ic-pagination-item.css",
+  shadow: true,
+})
+export class PaginationItem {
+  @Element() el: HTMLIcPaginationItemElement;
+
+  /**
+   * The current page number.
+   */
+  @Prop() page: number | null;
+  /**
+   * If true the pagination item is selected.
+   */
+  @Prop() selected: boolean = false;
+  /**
+   * The type of pagination item 'page' or 'ellipsis'.
+   */
+  @Prop() type: IcPaginationItemType;
+  /**
+   * If true the item is disabled.
+   */
+  @Prop() disabled: boolean = false;
+  /**
+   * The appearance of the pagination, e.g. dark or light. Default appearance if value not set.
+   */
+  @Prop() appearance: IcThemeForeground;
+  /**
+   * The label applicable when simple pagination is being used.
```

Comment:
Wondering if something like this might be clearer?
```suggestion
   * The label for the pagination item (applicable when simple pagination is being used).
```